src/openklant/conf/base.py

```diff
@@ -25,10 +27,12 @@
     "openklant.components.contactgegevens",
     # Django libraries
     "localflavor",
-    "django_structlog",
 ]
 
-MIDDLEWARE = MIDDLEWARE + ["openklant.utils.middleware.APIVersionHeaderMiddleware"]
+MIDDLEWARE += [
+    "openklant.utils.middleware.APIVersionHeaderMiddleware",
+    "csp.middleware.CSPMiddleware",
```

Comment:
Is this necessary? OAF also adds the middleware (ratelimited) https://github.com/maykinmedia/open-api-framework/blob/main/open_api_framework/conf/base.py#L408